### PR TITLE
Fix a typo in manage-ticdc-md

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Fix a typo in manage-ticdc-md.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_3.log --addr=0.0.0.0:830
 
 ## 使用 TiUP 升级 TiCDC
 
-本部分介绍如何使用 TiUP 来升级并开启 TiCDC 的集群。在以下例子中，假设需要将 TiCDC 和集群升级到 v4.0.6。
+本部分介绍如何使用 TiUP 来升级开启了 TiCDC 的集群。在以下例子中，假设需要将 TiCDC 和集群升级到 v4.0.6。
```
Comment (yikeke) [inline ticdc/manage-ticdc.md:66]:
```suggestion
本部分介绍如何使用 TiUP 来升级 TiCDC 集群。在以下例子中，假设需要将 TiCDC 组件和整个 TiDB 集群升级到 v4.0.6。
```

**Review by yikeke — COMMENTED** [review]
@overvenus PTAL at my suggestion:

**Review by amyangfei — COMMENTED** [review]
LGTM

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**